alacritty_terminal/src/renderer/mod.rs

```diff
@@ -1026,19 +1027,15 @@ impl<'a> RenderApi<'a> {
         };
 
         // Get font key for cell
-        // FIXME this is super inefficient.
-        let font_key = match (
-            cell.flags.contains(cell::Flags::BOLD),
-            cell.flags.contains(cell::Flags::ITALIC),
-        ) {
-            (false, false) => glyph_cache.font_key,
-            (true, false) => glyph_cache.bold_key,
-            (false, true) => glyph_cache.italic_key,
-            (true, true) => glyph_cache.bold_italic_key,
+        let font_key = match cell.flags | (Flags::BOLD & Flags::ITALIC) {
+            Flags::BOLD_ITALIC => glyph_cache.bold_italic_key,
```

Comment:
Shouldn't the match expression be `cell.flags & (Flags::BOLD | Flags::ITALIC)`?

Reply:
Or simply `cell.flags & Flags::BOLD_ITALIC`

Reply:
Completely correct, looks like I've twisted things around. I haven't used the `BOLD_ITALIC` flag since I added that later, but now that's also a great suggestion.